README.md

```diff
@@ -67,6 +71,20 @@ Deploy:
 kubectl kustomize config/default | kubectl apply -f -
 ```
 
+At this point you should see the `controlplane` and `dataplane` `Pods` running
+in the `blixt-system` namespace:
+
+```console
+$ kubectl -n blixt-system get pods
+NAME                                 READY   STATUS    RESTARTS   AGE
+blixt-controlplane-cdccc685b-9dxj2   2/2     Running   0          83s
+blixt-dataplane-brsl9                1/1     Running   0          83s
+```
+
+Check the `config/samples` directory for `Gateway` and `*Route` examples you
```

Comment:
```suggestion
Check the `config/samples` directory for `Gateway` and `Route` examples you
```

Reply:
This was intentional, in Gateway API we often refer to the various route types as `*Routes` or `xRoutes`.